### PR TITLE
feat: Add reference for binary car loan dataset

### PR DESCRIPTION
**Pull request by alexnanny**
Add reference documentation for the car loan dataset

### CODE REVIEW

**Comment by nnansters** [comment]
Looks good to me. Want to continue on this branch?

**Comment by alexnanny** [comment]
> Looks good to me. Want to continue on this branch?

Thanks @nnansters ! No, I have another branch for further docs updates

File: docs/datasets/binary_car_loan.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- `size_of_downpayment`: A categorical feature with 10 categories, representing the percentage in increments of 10%
+  of the size of the downpayment of the car value.
+
+- `tenure`: A numerical feature describing how many years the costumer has been driving.
+
+
+There are 3 columns that reference the output of the model:
+
+- `y_pred_proba`: The model predicted probability of the customer repaying the loan.
```
Comment (nikml) [inline docs/datasets/binary_car_loan.rst:62]: Can you double check on this. I think it may be the probability of the customer **not** repaying the loan.

Comment (alexnanny) [reply docs/datasets/binary_car_loan.rst:62]: Checking the `y_pred_proba` against the `repaid` target, it looks like the modes is predicting the customer repaying the loan

**Pull request merged**